03_letters/anishaNair/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	sortAndPrint(countLetters("aba"))
+}
+
+func countLetters(s string) map[rune]int {
+	elementMap := make(map[rune]int)
+	for _, c := range s {
+		_, ok := elementMap[c]
```

Comment:
Go's map access logic is surprisingly flexible. `elementMap[c]++` will work even before `elementMap[c]` has been initialized.
Try out this code on your machine
```go
func main() {
  m := map[int]int{}
  val, ok := map[0]
  fmt.Println(val, ok)
  m[0]++
  val, ok = m[0]
  fmt.Println(val, ok)
}
```

Reply:
Thank you!! fixed it up with putting just  elementMap[c]++